File: Disco.BLL/Models/Posts/CreatePostModel.cs
using AutoMapper;
using Disco.DAL.Entities;
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.Text;

namespace Disco.BLL.Models.Posts
{
    public class CreatePostModel
    {
        public string Description { get; set; }
        public List<IFormFile> PostImages { get; set; }
        public List<IFormFile> PostSongs { get; set; }
        public List<IFormFile> PostVideos { get;set; }
        public string Name { get; set; }
    }
}

File: Disco.Flutter/lib/data/network/network_models/create_post_model.dart
class CreatePostModel {
  String description;
  List<String> postImages;
  List<String> postSongs;
  List<String> postSongImages;
  List<String> postSongNames;
  List<String> postVideos;
  List<String> executorNames;

  CreatePostModel({
    required this.description,
    required this.postSongs,
    required this.postVideos,
    required this.postImages,
    required this.postSongImages,
    required this.postSongNames,
    required this.executorNames,
  });
}
